test/integration/bankHolidayAlert.js

```diff
@@ -49,20 +49,19 @@ describe('The bank holiday alert messaging', () => {
         .times(1)
         .reply(200, serviceApiResponse);
 
-      chai.request(server)
+      return chai.request(server)
```

Comment:
The use of promises in the tests looks good. Some of the chai.request() statements are returned and others not and I can't figure out why. Could you elaborate please.

Reply:
They are all returned, just some are done implicitly.
https://eslint.org/docs/rules/arrow-body-style

Reply:
Returning the promise is required https://mochajs.org/#working-with-promises

Reply:
I see, so it's the lack of the return of the 'chai.request()' promise that is causing the silent failure.
In that case, all tests should return the chai.request(), and the catch-throw isn't necessary so can be removed from the tests.

Reply:
Not quite as `chai.request` is being returned implicitly. I'll finish the changes to use async, it looks quite a bit better.

Reply:
That's the theory, but the 'unknown page' test below fails silently with no return from chai.request, but correctly reports the test as failing if you **do** return the value.